internal/pkg/service/buffer/sink/tablesink/storage/level/local/writer/disksync/config.go

```diff
@@ -39,18 +39,18 @@ type Config struct {
 	// Mode defines sync mode: more durable ModeDisk or faster ModeCache.
 	Mode Mode `json:"mode" configKey:"mode" validate:"required,oneof=disabled disk cache" configUsage:"Sync mode: \"disabled\", \"cache\" or \"disk\"."`
 	// Wait defines whether the operation should wait for sync.
-	Wait bool `json:"wait" configKey:"wait" validate:"excluded_if= Mode disabled" configUsage:"Wait for sync to disk OS cache or to disk hardware, depending on the mode."`
+	Wait bool `json:"wait" configKey:"wait" configUsage:"Wait for sync to disk OS cache or to disk hardware, depending on the mode."`
```

Comment:
In this config struct, additional values were forbidden.

It can cause probles, for example, we will modify global Stream API configruation and a Sink config patch will be no more valid.